app/views/offices/index.html.erb

```diff
@@ -0,0 +1,75 @@
+<%- @page_title = 'Locator Maps' -%>
+
+<div class="usa-grid usa-section usa-layout-docs">
+  <%= render 'shared/sidenav' %>
+
+  <div class="usa-width-three-fourths usa-layout-docs-main_content">
+    <h1><%= @page_title %></h1>
+
+    <p>Use the search form below to find local Transportation Offices (<%= abbr_tag('pppo') %>s) and Truck Weight Scales.</p>
+
+    <div id="geolocation-notice" hidden>
+      <p>You may also search for Transportation Offices and Truck Weight Scales using your device’s geolocation features. To use these features, you will be prompted to allow access to your location. We use the information provided by your device for the purposes of finding nearby Transportation Offices and Truck Weight Scales but <b>we do not store your location</b>.</p>
```

Comment:
What do you think about something like:

"You can also search for the closest Transportation Offices and Truck Weight Scales to your current location. To enable this feature, you will be prompted to allow move.mil access to your location. Your location is only used to find the nearest Transportation Offices and Truck Weight Scales, and <strong>move.mil will not store or remember your location</strong>."